.travis.yml

```diff
@@ -1,6 +1,23 @@
+sudo: required
+
 language: node_js
 node_js:
   - node
 
+services:
+  - docker
+
+before_install:
+  - pip install docker-compose
+
 script:
-  - yarn run travis
+  - docker-compose -f docker-compose-tests.yml build travis-tests
+  - docker-compose -f docker-compose-tests.yml run travis-tests
+
+after_success:
+  - docker-compose -f docker-compose-tests.yml build travis-tests-integration
```

Comment:
I see, 404 errors occur even though the file is present locally. Works OK when deployed. This is caused by symlinks between docker volumes and host folders. Removing the local output folder before build seems to resolve this. Will update the start script